misk/src/main/kotlin/misk/concurrent/ExplicitReleaseBlockingQueue.kt

```diff
@@ -37,6 +37,16 @@ class ExplicitReleaseBlockingQueue<T> internal constructor(
     return numReleased
   }
 
+  /** releases all items from the pending queue, returning the number of items released */
+  fun releaseAll() : Int {
+    var numReleased = 0
+    while (true) {
+      val e = pending.poll() ?: return numReleased
+      visible.add(e)
+      numReleased ++
```

Comment:
is it koitlin style to have a space between var and ++?

Reply:
strange